doc/nrf/installation/install_ncs.rst

```diff
@@ -48,6 +48,8 @@ Depending on your preferred development environment, install the following requi
 
       * The latest version of the :ref:`requirements_clt` package.
         Download it from the `nRF Command Line Tools`_ page.
+      * The universal version of SEGGER J-Link, which is required for SEGGER J-Link to work correctly with both Intel and ARM assemblies.
+        The installation prompt appears when you install the nRF Command Line Tools package.
```

Comment:
Note: if you just download the `.deb` file and double click it (installing it using package manager GUI), the promt does _not_ appear anywhere. This part confused me when trying to do a fresh setup of my environment.

Reply:
Same on Windows after I uninstalled all J-Links and the tools themselves.

Reply:
Rephrased based on the findings listed in https://nordicsemi.atlassian.net/browse/NRFU-1160?focusedCommentId=1936994